tensorboard/webapp/feature_flag/views/feature_flag_modal_trigger_component.ts

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {MatDialog, MatDialogRef} from '@angular/material/dialog';
+import {Store} from '@ngrx/store';
+import {State} from '../../app_state';
+import {featureFlagOverrideChanged} from '../actions/feature_flag_actions';
+import {
+  getFeatureFlags,
+  getShowFlagsEnabled,
+} from '../store/feature_flag_selectors';
+import {FeatureFlagPageContainer} from '../views/feature_flag_page_container';
+
+@Component({
+  selector: 'feature-flag-modal-trigger-component',
+  template: ``,
+  styles: [],
+})
+export class FeatureFlagModalTriggerComponent {
```

Comment:
Name this "FeatureFlagModalTriggerContainer". 

Our naming convention: Typically any Angular Component that gets data from ngrx is named with "Container" rather than with "Component". Angular Components with "Component" in the name typically are ngrx-free - if they need data from ngrx they then are wrapped by a "Container" layer to do the ngrx stuff.

Reply:
Thanks for the detailed explanation, that makes a lot of sense.